editor.planx.uk/src/@planx/components/FileUploadAndLabel/Public.tsx

```diff
@@ -307,7 +307,7 @@ const InteractiveFileListItem = (props: FileListItemProps) => {
           aria-haspopup="dialog"
           size="small"
         >
-          <span>Help</span>
+          <span>Info</span>
```

Comment:
_Non-blocking Question_

I was really curious about this but I could never find an instances where it says "Info" as I played around in the editor/previews 😅 

If it's not too much of a hassle would you be able to point me at this @ianjon3s? It's a silly request I'm just curious about where this lives as I haven't encountered it and I feel like I'm missing something.

Reply:
Hey @Mike-Heneghan ,

The help/info text is displayed on the Upload + Label component for files that have further info text added.

Here's an example:
https://2435.planx.pizza/testing/upload-label-logic-example/preview?analytics=false

There was a decision a while back from user-testing that we should rename this to read "Info", I failed to mention this in the original PR description.

Reply:
Amazing, thanks for the explanation and the example link 🙌